torchdynamo/debug_utils.py

```diff
@@ -646,24 +762,42 @@ def debug_wrapper(gm, example_inputs, **kwargs):
             config.raise_on_backend_error = True
             if config.repro_level == 3:
                 dump_to_minify_after_dynamo(gm, example_inputs, compiler_name)
-            try:
+
+            # Check for either accuracy (level 4) or other type of failures.
```

Comment:
Would be nice if we could share more code between the two, but not a big deal if it's awkward.